flojoy

```diff
@@ -352,6 +356,11 @@ sleep 1
 # start project services
 if [ -n "$venvCmd" ]; then
    $venvCmd && npm run start-project
+
+elif $test_ci; then
```

Comment:
This can be written as follows:
```
if $test_ci; then
     start_project_cmd=npm run start-project:ci
else 
      start_project_cmd=npm run start-project
fi
 # start project services
if [ -n "$venvCmd" ]; then
   $venvCmd && $start_project_cmd
else
    $start_project_cmd
fi
```

Reply:
$test_ci variable check should be checked independently.